src/pages/patientView/genomicOverview/Tracks.tsx

```diff
@@ -78,29 +79,27 @@ export default class Tracks extends React.Component<TracksPropTypes, {}> {
 
 
         // --- mutation events bar chart ---
-        _.each(this.props.sampleManager.samples, (sample: ClinicalDataBySampleId) => {
-            if (_.includes(uniqMutSampleIds, sample.id)) {
-                var _trackData = _.filter(this.props.mutations, function (_mutObj: any) {
-                    return _mutObj.sampleId === sample.id;
-                });
-                tracksHelper.plotMuts(paper, config, chmInfo, rowIndex, _trackData, sample.id);
-                rowIndex = rowIndex + 1;
-
-                if (this.props.sampleManager.samples.length > 1) {
-                    const id = `#mutTrack${sample.id}`;
-                    const $container = $(id);
-                    const pos = {x: parseInt($container.attr('x')) - 10, y: parseInt($container.attr('y')) - 5};
-                    const $newContainer = $(`<svg id="${id}" height="12" width="12" />`);
-                    $newContainer.attr(pos);
-                    $container.replaceWith($newContainer);
-
-                    let comp: any = this.props.sampleManager.getComponentForSample(sample.id);
-
-                    ReactDOM.render(
-                        comp,
-                        $newContainer[0]
-                    );
-                }
+        _.each(sequencedSamples, (sample: ClinicalDataBySampleId) => {
+            var _trackData = _.filter(this.props.mutations, function (_mutObj: any) {
```

Comment:
all the code here is the same except for the top line (the `_.each`) - everythings marked different because by removing the redundant `if`, now that Im computing `sequencedSamples`, the indentation goes left by one.